docs/ReleaseNotes.md

```diff
@@ -1,6 +1,7 @@
 # Release Notes
 
 ## Difference between the latest release and master
+- Enables vehicle entity yeild to merging entity. See also [this video.](https://user-images.githubusercontent.com/10348912/128287863-8a2db025-d1af-4e54-b5a3-08e5d2e168e4.mp4)
```

Comment:
```suggestion
- Enables vehicle entity yield to merging entity. See also [this video.](https://user-images.githubusercontent.com/10348912/128287863-8a2db025-d1af-4e54-b5a3-08e5d2e168e4.mp4)
```

Reply:
Thank you!
I fixed in this PR.
https://github.com/tier4/scenario_simulator_v2/pull/433/commits/0b3b3d5cf7bc1d58308b906f2fb01a1a4e71e785